mbuild/compound.py

```diff
@@ -712,7 +712,7 @@ def label_rigid_bodies(self, discrete_bodies=None, rigid_particles=None):
 
         >>> import mbuild as mb
         >>> from mbuild.utils.io import get_fn
-        >>> benzene = mb.load(get_fn('benzene.mol2'))
+        >>> benzene = mb.get_fn(get_fn('benzene.mol2'))
```

Comment:
```suggestion
        >>> benzene = mb.load(get_fn('benzene.mol2'))
```

I think this change was by mistake?